src/toil/common.py

```diff
@@ -524,3 +530,30 @@ def parseSetEnv(l):
             raise ValueError('Empty name')
         d[k] = v
     return d
+
+
+def getToilWorkflowDir(workflowID, configWorkDir=None):
+    """
+    Returns a path to the directory where worker directories and the cache will be located for this
+    workflow.
+    :param str workflowID: Unique identifier for the workflow
+    :param str configWorkDir: Value passed to the program using the --workDir flag
+    :return: Path to the workflow directory
+    :rtype: str
+    """
+    workDir = configWorkDir or os.getenv('TOIL_WORKDIR') or tempfile.gettempdir()
+    if not os.path.exists(workDir):
+        raise RuntimeError("The directory specified by --workDir or TOIL_WORKDIR: %s " % workDir +
```

Comment:
Also, a grammar purist would want the message to be `The directory specified by --workDir or TOIL_WORKDIR (%s) does not exist`. It's good to have delimiters around an interpolated string so as to better identify empty strings.